pkg/proxy/iptables/proxier.go

```diff
@@ -121,8 +121,12 @@ type serviceInfo struct {
 	servicePortChainName     utiliptables.Chain
 	serviceFirewallChainName utiliptables.Chain
 	serviceLBChainName       utiliptables.Chain
+
+	localWithFallback bool
 }
 
+const localWithFallbackAnnotation = "traffic-policy.network.alpha.openshift.io/local-with-fallback"
```

Comment:
there is going to be an "official" annotation, `service.kubernetes.io/topology-aware-routing`
https://github.com/kubernetes/enhancements/tree/master/keps/sig-network/2433-topology-aware-hints#configuration

do we want to make it more official?
```
service.kubernetes.io/topology-aware-routing = local-with-fallback
```